wordpress-mobile/WordPress-iOS
# Show announcements in App Settings even if it's a fresh install

Fixes #15152

To test:

- Delete the app from the device you are using
- Manually change the version in this branch to 50.3
- Launch and verify that no announcement screen is shown at app launch
- Verify that in "App Settings" you see "What's New in WordPress" and when tapping it, the announcement screen appears (see example below)
- Discard the changes
- Delete the app again
- Checkout and run a previous version (e.g. `develop`)
- Checkout this branch again
- Manually change the version again to 50.3
- Verify that the announcement screen (see below) appears at startup, and it's also available in "App Settings/What's New in WordPress"

example screenshot of announcements

<p align=center>
<img height = 500 src=https://user-images.githubusercontent.com/34376330/102263107-867b8780-3ed9-11eb-86b7-a84538abc85b.PNG>
</p>

PR submission checklist:
